proposals/lip-0043.md

```diff
@@ -210,6 +210,9 @@ def verify(trs: Transaction) -> None:
     if not all(validatorKeys[i] < validatorKeys[i + 1] for i in range(len(validatorKeys) - 1)):
         raise Exception("Validator keys are not in lexicographic order.")
 
+    if len(trs.params.sidechainValidators) < 1 or len(trs.params.sidechainValidators) > MAX_NUM_VALIDATORS:
```

Comment:
the same check already exists, (lines 201-202 above), am I right?

Reply:
removed